src/pages/CertificatesViewAll.js

```diff
@@ -54,40 +61,47 @@ export default function CertificatesViewAll() {
         {data && (
           <Table
             action={(item) =>
-              navigate(`/certificate/${item.original_token_id}`)
+              navigate(`/certificate/${item.original_token_id || item.id}`)
             }
             headers={headersMap[current]}
             rows={data.map((cert) => [
               /* TODO with other cert viewing stories
                   - create rows/cels per persona using "headersMap as an example"*/
               <NameCell
                 key={cert.id}
-                date={new Date(cert.created_at).toISOString()}
-                name={`SQNC-HYPROOF-${cert.original_token_id.toString().padStart(4, '0')}`}
+                date={formatTimelineDate(cert.created_at)}
+                name={formatName(cert)}
               />,
-              `${cert.hydrogen_quantity_wh}`,
-              `${cert.energy_consumed_wh}`,
-              `${cert.embodied_co2}`,
-              stateToStatus[(cert.embodied_co2 && 'co2') || cert.state],
+              `${cert.hydrogen_quantity_wh / 1000000} MWh`,
+              `${cert.energy_consumed_wh / 1000000} MWh`,
+              cert?.embodied_co2 ? `${cert.embodied_co2} g CO2e` : '',
```

Comment:
can we convert this to kg please in line with @dc-andysign's open PR #67 . Just reduces the odds of us missing the conflict

Reply:
This has been update, pardon me also added unit similar to andies, but used `toFixed`